### PR TITLE
Add support for django 1.10

### PR DESCRIPTION
**Pull request by meshy**
The tests currently fail on django 1.10.

### CODE REVIEW

**Comment by Ian-Foote** [comment]
I just had a look at this and it seems the big blocker is `django-sirtrevor`, which hasn't been updated for 2 years. Do we want to fork that, or use something else?

**Comment by meshy** [comment]
I think I'd actually be tempted to remove `django-sirtrevor` completely for the moment, and focus on the dynamic routes side of things.

I've started work on an alternative, [`django-sirtrevor-field`](https://github.com/peroshi/django-sirtrevor-field), but it's not yet ready, as I'm a little stuck on selenium tests. Once that's working, it might make sense to re-make the pages based on that, but I think it might be better as a seperate library (or even just an example in the readme... should be super simple.

File: .travis.yml [hunk 1]
```diff
@@ -12,12 +13,10 @@ env:
   matrix:
     - DJANGO='django~=1.8.0'
     - DJANGO='django~=1.9.0'
-    - DJANGO='django==1.10b1'
+    - DJANGO='django==1.10.0'
```
Comment (meshy) [inline .travis.yml]: Fixed with a rebase.

**Comment by meshy** [comment]
I've removed django-sirtrevor in #84

**Pull request closed without merging**